compiler/src/safety/safetyAbsExpr.ml

```diff
@@ -114,8 +114,10 @@ let op2_to_abs_binop op2 = match op2 with
   | E.Odiv _ -> AB_Arith Texpr1.Div
 
   | E.Olsr _ -> AB_Wop (Wshift Unsigned_right)
-  | E.Olsl _ -> AB_Wop (Wshift Unsigned_left)
-  | E.Oasr _ -> AB_Wop (Wshift Signed_right)
+  | E.Olsl (Op_w _) -> AB_Wop (Wshift Unsigned_left)
+  | E.Olsl Op_int -> AB_Unknown
+  | E.Oasr (Op_w _) -> AB_Wop (Wshift Signed_right)
+  | E.Oasr Op_int -> AB_Unknown
```

Comment:
Why do we need to distinguish `Op_w` and `Op_int` here? And not for other operations?

Reply:
I don’t know. Could you provide an example in which not doing this distinction is a problem?

Reply:
No, I'm trying to understand what you did.